pages/AWorkedExample.html

```diff
@@ -170,7 +170,7 @@ <h4>Example: Adding a response to our Sample Exploration</h4>
     exploration, let's go to the second card (Method 2) and see what answers
     learners have been trying. Here's an example of what this may look like:
   </p>
-  <img src="static/images/tutorial/unhandledAnswers.png" class="tutorial-image">
+  <img src="static/images/tutorial/unhandledAnswers.png" class="tutorial-image" alt="A list of all of Oppia's responses, followed by an 'Add New Oppia Response Button', below which is a list of all answers the learners have been trying for a particular exploration.">
```

Comment:
below which is a --> followed by 

list of all answers ... --> list of answers that have been submitted for this card.

(Note that the list of answers is only for the given card, not the full exploration.)

Reply:
Apologies for the misunderstanding. Thankyou for the feedback!